redis/src/cluster_async/mod.rs

```diff
@@ -1355,6 +1380,24 @@ where
                         },
                     }));
                 }
+                Next::RetryBusyLoadingError {
```

Comment:
I'm not near the PC so I might be wrong, but it seems to me that we don't really enforce retries if the loading error is repeated- we get a loading error and we create a new future. If this request will face another loading error afterwards (let's say if it was a primary and it is very likely that it's still loading), we will just create a new future with full number of retries and queue it into the in flights requests.
Also, do we want to have a sleep before we retry the request, if it is a primary?